src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      const response = await getPeople();
+
+      setPeople(response);
+    };
+
+    try {
+      fetchPeople();
+    } catch (error) {
+      throw new Error(`${error}`);
```

Comment:
and you don't catch this `Error` whatsoever, create stateful value e.g. `errorText` and use conditional rendering to render your errorText